ios/FluentUI.Demo/FluentUIDemoTests/IndeterminateProgressBarTest_SwiftUI.swift

```diff
@@ -5,16 +5,31 @@
 
 import XCTest
 
-class IndeterminateProgressBarTestSwiftUI: BaseTest {
-    override var controlName: String { "IndeterminateProgressBar" }
-
+class IndeterminateProgressBarTestSwiftUI: IndeterminateProgressBarTest {
     override func setUpWithError() throws {
         try super.setUpWithError()
         app.staticTexts["SwiftUI Demo"].tap()
     }
 
     // launch test that ensures the demo app does not crash and is on the correct control page
-    func testLaunch() throws {
+    override func testLaunch() throws {
         XCTAssertTrue(app.navigationBars.element(matching: NSPredicate(format: "identifier CONTAINS %@", controlName)).exists)
     }
+
+    override func testStartStopHide() throws {
+         let animatingSwitch: XCUIElement = app.switches["Animating"]
+         let hidesWhenStoppedSwitch: XCUIElement = app.switches["Hides when stopped"]
+
+         hidesWhenStoppedSwitch.tap()
+         XCTAssert(indeterminateProgressBarExists(status: inProgress))
+
+         animatingSwitch.tap()
+         XCTAssert(indeterminateProgressBarExists(status: progressHalted))
+
+         hidesWhenStoppedSwitch.tap()
+         XCTAssert(!indeterminateProgressBarExists(status: inProgress))
```

Comment:
Thinking through what's happening in the test - does it make sense to assert !indeterminateProgressBarExists(status: progressHalted) as well?